adapters/beachfront/beachfront.go

```diff
@@ -67,8 +69,8 @@ type BeachfrontVideoDevice struct {
 // ---------------------------------------------------
 //              Banner
 // ---------------------------------------------------
-type BeachfrontBannerRequest struct {
-	Slots          []BeachfrontSlot `json:"slots"`
+type beachfrontBannerRequest struct {
```

Comment:
This is also marshaled, so should not be private

Reply:
Resolving this as above.